bin/Rules.mk

```diff
@@ -11,8 +15,12 @@ DISTCLEAN += $(wildcard $(d)/gx-v*) $(wildcard $(d)/gx-go-v*) $(d)/tmp
 PATH := $(realpath $(d)):$(PATH)
 
 $(TGTS_$(d)):
-	rm -f $@
-	ln -s $(notdir $^) $@
+	rm -f $@$(binpostfix)
+ifeq ($(OS),Windows_NT)
```

Comment:
Same with this, use the variable exported here: https://github.com/ipfs/go-ipfs/blob/master/mk/util.mk#L4